arthur_bench/scoring/summary_quality.py

```diff
@@ -59,8 +64,21 @@ class SummaryQuality(Scorer):
     Comprehensive measure of summarization quality compared to a reference summary.
     """
 
-    def __init__(self):
-        self.evaluator = LLMChain(llm=ChatOpenAI(temperature=0), prompt=COMPARE)
+    def __init__(
```

Comment:
could you move the creation of the default values `llm` and the tokenizer to inside of the init?

ex.
```
llm = None
):
if llm is None:
    llm = ChatOpenAI(temperature=0)
```

Reply:
Moved those two values, retyped them as Optional values so the default `None` on them doesn't make a type checker angry. I left the context_window default value lookup piece in the signature though, doing a lookup for an int should be quite cheap.